x-pack/plugins/code/public/style/_utilities.scss

```diff
@@ -7,5 +7,5 @@
 }
 
 .codeMargin__title {
-  margin: $euiSizeXS 0 $euiSize;
+  margin: $euiSizeXS 0 $euiSize $euiSize;
```

Comment:
Did you mean to add margin to the left of the title?

<img src="https://d.pr/free/i/wkyQll+" width="50%" />

Reply:
I did, but apparently my eyes aren't what they used to be. Nice catch.